app/src/components/card/cardImage.tsx

```diff
@@ -0,0 +1,64 @@
+import Image from "next/image";
+import React from "react";
+import { useEffect, useRef, useState } from "react";
+import { isCollectionCardDataType } from "./card";
+
+const CardImage = ({ record }) => {
+  const [imageHeight, setImageHeight] = useState(144);
+  const imageRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    const updateImageHeight = () => {
+      if (imageRef.current) {
+        const image = imageRef.current as HTMLElement;
+        const imageWidth = image.offsetWidth;
+
+        setImageHeight(0.5 * imageWidth);
+      }
+    };
+    updateImageHeight();
+    window.addEventListener("resize", updateImageHeight);
+
+    return () => window.removeEventListener("resize", updateImageHeight);
+  }, []);
+
+  const [imageSrc, setImageSrc] = useState(
+    record.imageID ? record.imageURL : "/noImage.png"
+  );
+  const isCollection = isCollectionCardDataType(record);
+  return (
+    <div
+      ref={imageRef}
+      style={{
+        overflow: "hidden",
+        height: imageHeight,
+      }}
+    >
+      <Image
+        src={imageSrc}
+        alt=""
```

Comment:
There might be some occasions where alt text is needed and images are not decorative, so include this as a prop, perhaps from `record`'s title?

Reply:
Actually, you might need a new prop that is something along the lines of `isDecorative`. If it's false, then include the alt text, otherwise it should be empty. So far, all images we've encountered that this component can render has been decorative, right? If so, this is not needed right now.

Reply:
Yes this was the approved alt text for all card images so far. Will change once Clare gives the go ahead